doc/developers/maintainer.rst

```diff
@@ -57,81 +57,117 @@ permissions given to maintainers, which includes:
 Preparing a release PR
 ......................
 
+Major version
+~~~~~~~~~~~~~
+
 Releasing the first RC of e.g. version `0.99` involves creating the release
 branch `0.99.X` directly on the main repo, where `X` really is the letter X,
-**not a placeholder**. This is considered the *feature freeze*. The
-development for the major and minor releases of 0.99 should
-**also** happen under `0.99.X`. Each release (rc, major, or minor) is a tag
-under that branch.
+**not a placeholder**. The development for the major and minor releases of 0.99
+should **also** happen under `0.99.X`. Each release (rc, major, or minor) is a
+tag under that branch.
+
+This is done only once, as the major and minor releases happen on the same
+branch:
+
+   .. prompt:: bash $
+
+     git checkout -b 0.99.X
```

Comment:
The explanation for this command now misses the key point that the release manager needs to do it either under a different clone from the main repo, or under the same clone, but from and to `upstream` instead of `origin`.

Reply:
I expanded the commands to be more explicit.